doc/commands/distro-sync.8.rst

```diff
@@ -56,6 +56,8 @@ Options
 
 .. include:: ../_shared/options/from-repo.rst
 
+.. include:: ../_shared/options/from-vendor.rst
```

Comment:
Did you forget to commit `from-vendor.rst`?

Reply:
Yes, I did. I have added the missing file now.